From 8e1885bbccaae7403a09110894f7bf61c71a41b3 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 19 Dec 2019 23:36:19 +0200
Subject: [PATCH] [SW-0] preboot: remove anymore unused define

Since we removed the PLL clock switch glitch W/A, the PREBOOT_VER_BASE
define is not used anymore.
Finally remove it.

Change-Id: Id7e67834fdafe85766f45122d3b6e095339ad0cc
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/include/preboot.h b/include/preboot.h
index 7b03773..fbd9a68 100644
--- a/include/preboot.h
+++ b/include/preboot.h
@@ -25,7 +25,6 @@
 #define SPL_TEXT_BASE				0x7FF0080000ull
 #endif
 
-#define PREBOOT_VER_BASE			(SPL_TEXT_BASE - 0x1000)
 #define PREBOOT_SIZE		0x4000		/* 16KB */
 
 #define SPL_SIZE		0x10000		/* 64KB */
